crates/fuel-core/src/graphql_api/worker_service.rs

```diff
@@ -631,6 +676,7 @@ where
     }
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
Maybe it makes sense to pass `&ConsensusParameters` since now we have 2 parameters from it, in this case you can remove `too_many_arguments`

Reply:
Updated in dd03f921f3aa7b2aa861627fa62dc9f6f5ee8401.
Soon we might need some _POD_ structure to carry these parameters.